### PR TITLE
Add `is_alphanumeric` attribute to Autolink and `Repository.create_autolink`

### PR DESCRIPTION
**Pull request by janssonoskar**
Added _is_alphanumeric_ to Autolink to be able to choose between alphanumeric (default) and numeric

### CODE REVIEW

File: github/Repository.py [hunk 1]
```diff
@@ -992,20 +992,20 @@ def compare(self, base, head):
             self._requester, headers, data, completed=True
         )
 
-    def create_autolink(self, key_prefix, url_template):
+    def create_autolink(self, key_prefix, url_template, is_alphanumeric=True):
```
Comment (trim21) [inline github/Repository.py:995]: please use `NotSet` as default value

**Review by trim21 — CHANGES_REQUESTED** [review]

File: github/Repository.py (hunk 1 above)
Comment (janssonoskar) [reply github/Repository.py]: Fixed

**Comment by janssonoskar** [comment]
@EnricoMi Hi, I haven't got authentication to work to record tests, is that something you could help me with?

**Review by EnricoMi — APPROVED** [review]
LGTM!

**Pull request merged**